unison-cli/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -727,22 +727,22 @@ deleteGen suffix target mkTarget =
         )
 
 delete :: InputPattern
-delete = deleteGen Nothing "term or type" (DeleteTarget'TermOrType DeleteOutput'NoDiff)
+delete = deleteGen Nothing exactDefinitionTermQueryArg "term or type" (DeleteTarget'TermOrType DeleteOutput'NoDiff)
```

Comment:
This one maybe needs a TermOrTypeQueryArg

Reply:
Updated!